src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -1081,5 +1081,44 @@ class C { }
             Assert.True(tree.TryGetRoot(out var rootFromTryGetRoot));
             Assert.Same(rootFromGetRoot, rootFromTryGetRoot);
         }
+
+        [Fact]
+        // PROTOTYPE: checks we're dropping the incremental drivers for now, and running dual drivers as ISourceGenerator
```

Comment:
nit: typically `// PROTOTYPE` comments include the feature name, e.g. `// PROTOTYPE(source-generators)`